spec/rails_helper.rb

```diff
@@ -26,10 +26,18 @@
 end
 
 # Globally stub smartystreets
-stub_request(:any, /smartystreets.com/).to_return(
-  body: File.read('spec/fixtures/webmock_responses/smartystreets.json'),
-  status: 200
-)
+SMARTYSTREETS_STUB = \
+  stub_request(:any, /smartystreets.com/).to_return(
+    body: File.read('spec/fixtures/webmock_responses/smartystreets.json'),
+    status: 200
+  )
+
+def without_stub_request(stub)
+  wmsr = WebMock::StubRegistry.instance
+  wmsr.remove_request_stub(stub)
+  yield
+  wmsr.register_request_stub(stub)
+end
```

Comment:
This is pretty slick, but think it would be better if we figured out a way to accomplish something similar without hitting the API when specs are run. @nvick and I have been discussing taking VCR out of the project since it's not really being used.

Some other approaches that might work:
- using Geocoder stubbing like you suggested
- adding a smarty_streets_failure.json fixture that contains an error response from the API, then we could just stub using the failure json for specs that need it
- use VCR - I don't know how hard this would be and like I said, we've been talking about getting rid of it, so this is least favorable option in my mind at the moment. That said, if VCR were working properly and being used, I'm not opposed to keeping it around :)